pages/contributing-guide.md

```diff
@@ -0,0 +1,109 @@
+* Having a general understanding of [Linked Data](https://www.w3.org/DesignIssues/LinkedData.html). good [source](http://www.linkeddatatools.com/semantic-web-basics) to study linked data.
+* Having a general understanding of [JSON-LD](https://dl.acm.org/citation.cfm?id=2307827). A [video](https://www.youtube.com/watch?v=vioCbTo3C-4) for intuitive introduction of JSON.
+* Read these two papers [Hydra: A Vocabulary for Hypermedia-Driven Web APIs
+(Markus Lanthaler, Christian Gütl (2013))](http://www.markus-lanthaler.com/research/hydra-a-vocabulary-for-hypermedia-driven-web-apis.pdf) and [Creating 3rd Generation Web APIs with Hydra
+(Markus Lanthaler (2013))](http://www.markus-lanthaler.com/research/creating-3rd-generation-web-apis-with-hydra.pdf)
+* Read the old [HYDRA Draft](https://www.hydra-cg.com/spec/latest/core/) and [updated specifications](https://github.com/HydraCG). *Very important* is the current work done for [Draft's use-cases](https://github.com/HydraCG/Specifications/tree/master/drafts/use-cases)
+* Read this seminal [paper about Web APIs](https://arxiv.org/abs/1609.07108). This [other paper](https://arxiv.org/pdf/1809.01622.pdf) explains different use cases for graphs applied to databases.
+* Read [Hydra-CG gitbook](https://github.com/HydraCG/gitbook)
+* Read [hydrus WIKI](https://www.hydraecosystem.org/00-Home). Run `hydrus` and try to set up a basic API. Read the [documentation](http://hydrus.readthedocs.io/en/latest/)
+* If you want to have fun with a more dynamic demo, run [hydra-flock-demo](https://github.com/HTTP-APIs/hydra-flock-demo)
+* Join us on [slack](https://join.slack.com/t/hydraecosystem/shared_invite/enQtNzM3NTg5NzQ2MDUxLWU1MjM3ZGRhZWM4ZTg1ODBjMTljNTQwNzAwMGM3ZDlmYTY3Y2E4OGJmN2NlZWRjMWIzY2MzN2NjOTIyYmQ1ZjU). There are different channels in Slack in which to interact with other contributors:
+    - [#general](https://app.slack.com/client/TMGNKBP5X/CMR9RFB0E): talk about anything related to Hydra Ecosystem
+    - [#hydrus](https://app.slack.com/client/TMGNKBP5X/CMR0C8WJ3): discuss issues, solutions and ideas for hydrus
+    - [#tech-writing](https://app.slack.com/client/TMGNKBP5X/CNBS844FL): suggestions related to documentation, website content can be shared here
+* Read the other blogposts in our [Medium Publication](https://medium.com/w3c-hydra-development-community)
+* Get familiar with [Docker container engine](https://docker-curriculum.com/) and its best practices to deploy virtualized environments.
+
+## Community values
+
+In the interest of fostering an open and welcoming environment, contributors and maintainers pledge to make participation in our project and our community a harassment-free experience for everyone - regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
```

Comment:
You literally just copied the Tensorflow community values content without bothering to change the name.